build/CoreFoundation/dll/CoreFoundation.vcxproj

```diff
@@ -126,7 +126,7 @@
       <ModuleDefinitionFile>CoreFoundation.def</ModuleDefinitionFile>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AdditionalDependencies>mincore.lib;libxml2.lib;icudt.lib;icuin.lib;icuuc.lib;libdispatch.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories>$(StarboardBasePath)\Frameworks\limbo;$(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(StarboardBasePath)\Frameworks\limbo;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
```

Comment:
yeah don't worry about it. There has been a lot of cleanup of the old code. You don't know half of the horrors of this project ;-)